src/terra-dev-site/app/utilities.2/Logger.app.mdx

```diff
@@ -0,0 +1,47 @@
+# Logger
+
+`terra-application` provides an override-able logger to handle error, info, and warning messages.
+
+## Usage
+
+The logger works like the `console` for the `warn`, `error`, and `info` methods. By default the logger will send the messages to the corresponding console methods and no-op in production for the info and warning messages.
+
+```js
+import logger from 'terra-application/lib/utils/logger';
+
+logger.error('error');
+logger.warn('warning');
+logger.info('info');
+```
+
+## Overriding
+
+It may be desirable to override the default implementation of the logger. For example, logs could be sent to a server side log aggregator. The logger is a singleton and should only be overridden once per application. To override the logger implementation, use the `initializeLogger` function and provide a new function for the logger. All loggers must have a method signature of `(obj1 [, obj2, ..., objN])`, the same as the [console.warn](https://developer.mozilla.org/en-US/docs/Web/API/Console/warn), [console.error](https://developer.mozilla.org/en-US/docs/Web/API/Console/error), and [console.info](https://developer.mozilla.org/en-US/docs/Web/API/Console/info) methods.
+
+```js
+import { initializeLogger } from 'terra-application/lib/logger';
+
+initializeLogger({
+  onInfo: (...args) => console.log('[Info] ', ...args),
+  onWarn: (...args) => console.log('[Warn] ', ...args),
```

Comment:
When overriding, do consumers need to add in the no-opt logic for log & warning messages?
```suggestion
  onInfo: (...args) => console.log('[Info] ', ...args),
  onWarn: (...args) => console.warn('[Warn] ', ...args),
```